interface/forms/newpatient/common.php

```diff
@@ -438,6 +441,9 @@ function cancelClickedOld() {
                                         $userService = new UserService();
                                         $users = $userService->getActiveUsers();
                                         foreach ($users as $activeUser) {
+                                            if ($activeUser->getAuthorized() !== true) {
+                                                continue;
+                                            }
                                             echo "<option value='" . attr($activeUser->getId()) . "'";
```

Comment:
nice, the entity has the `getAuthorized()` method

Reply:
This service is fully hydrated where all columns have getter/setters